src/components/Items/Items.jsx

```diff
@@ -3,7 +3,6 @@ import React from 'react';
 
 export default (props) => {
   return <div className="Menu_Items">
-    <Items name={props.items.name} />,
-    <Items price={props.items.price} />
+    <p> {props.name}, {props.price} </p>
```

Comment:
Here, you were trying to use "Items" but this is in `Items.jsx`. Essentially you were on the right track, but you can simply just return a single div or something containing the information we need.